src/Microsoft.Azure.SignalR.Common/ClientInvocation/IClientResultsManager.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NET7_0_OR_GREATER
+using System;
+using Microsoft.Azure.SignalR.Protocol;
+using System.Threading.Tasks;
+using System.Threading;
+using System.Buffers;
+using Microsoft.AspNetCore.SignalR.Protocol;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal interface IClientResultsManager
+    {
+        public ulong GetNewInvocation();
+
+        public void AddServiceMappingMessage(string invocationId, ServiceMappingMessage serviceMappingMessage);
+
+        public void RemoveServiceMappingMessageWithOfflinePing(string instanceId);
```

Comment:
Looks it's doing cleanups that not simply remove mapping info. Rename to `CleanupInvocations()`?

Reply:
fixed